lib/daylight.cpp

```diff
@@ -1193,14 +1336,16 @@ bool Daylight::simStandard(int blindGroupNum, int setting, Control *model){
 
         //dctimestep | rcollate for the sensor for the sun
         //Added this line from an email that didn't exist before
-        arguments[2]=sensorSunDC;
-        arguments[3]=sunSMX;
+        arguments[0]=sensorDirSunDC;
+        arguments[1]=sunSMX;
```

Comment:
These two lines look suspect to me. Nearly all the other usage of this `arguments` vector is different from this one, and it looks like this usage assumes there are two elements in the vector and maybe no more. So this looks pretty unsafe with all of the `arguments.clear()` calls.